webclient/webclient/src/main/java/io/helidon/webclient/WebClientRequestBuilderImpl.java

```diff
@@ -147,7 +149,8 @@ private WebClientRequestBuilderImpl(LazyValue<NioEventLoopGroup> eventGroup,
         this.services = configuration.clientServices();
         this.readerContext = MessageBodyReaderContext.create(configuration.readerContext());
         this.writerContext = MessageBodyWriterContext.create(configuration.writerContext(), headers);
-        Context.Builder contextBuilder = Context.builder().id("webclient-" + REQUEST_NUMBER.incrementAndGet());
+        this.requestId = REQUEST_NUMBER.incrementAndGet();
```

Comment:
If we want to have request id, it should be configurable (esp. when using tracing, we may want to bind it to the trace id).
Also this is usually handled in logging through MDC (Mapped diagnostic context) in frameworks that support such a feature.
I am not sure the chosen approach is aligned with overall logging in Helidon (e.g. using (..) and message).

Reply:
fixed